pkg/exchange/kucoin/exchange.go

```diff
@@ -13,11 +13,13 @@ import (
 	"golang.org/x/time/rate"
 
 	"github.com/c9s/bbgo/pkg/exchange/kucoin/kucoinapi"
-	"github.com/c9s/bbgo/pkg/types"
 	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
 )
 
 var marketDataLimiter = rate.NewLimiter(rate.Every(500*time.Millisecond), 1)
+var queryTradeLimiter = rate.NewLimiter(rate.Every(5*time.Second), 1)
+var queryOrderLimiter = rate.NewLimiter(rate.Every(5*time.Second), 1)
```

Comment:
5 sec / req might be too wide?
https://m.kucoin.com/news/en-adjustment-of-the-spot-and-futures-api-request-limit?utm_source=redditinfo

Reply:
oh you're right

Reply:
updated